R/pagination.R

```diff
@@ -5,9 +5,9 @@ extract_link <- function(gh_response, link) {
   if (is.null(links)) {
     return(NA_character_)
   }
-  links <- trim_ws(strsplit(links, ",")[[1]])
+  links <- trimws(strsplit(links, ",")[[1]])
```

Comment:
I would stay with our own `trim_ws`. R's `trimws` is slightly broken, it does not handle all unicode space characters. It would also require R >= 3.2.0.